site/code-snippets/web5/build/decentralized-web-nodes/query-from-dwn.js

```diff
@@ -64,7 +64,40 @@ export async function queryRecordWithParentId(web5) {
   return response;
 }
 
-export async function queryFromDwnByPathAndType(web5) {
+export async function queryFromDwnByPathAndStructure(web5) {
+
+  const musicProtocolDefinition = {
+    protocol: "https://music.org/protocol",
```

Comment:
```suggestion
    protocol: "https://music.org/protocol",
    published:true,
```
The protocol won't install without the `published` property being defined